participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/UserRegistrationForm.java

```diff
@@ -55,6 +55,7 @@
   private boolean tempPassword;
   private LocalDateTime tempPasswordDate;
   private String appId;
+  private String appName;
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Fixed review comment.